README.md

```diff
@@ -13,6 +15,8 @@
  ]
 ```
 
-# Method
+2. `composer require danhariton/data-boxes`
+
+## Usage
 
-* LoadData(ICO) get information about data box by ICO.
\ No newline at end of file
+* Use [`\DanHariton\DataBoxes\Service\DataBoxInterface::loadData(string $ico): array`](./src/Service/DataBoxInterface.php) to get information about data box by VAT (IČO).
```

Comment:
Use absolute path to called method including link to source file to make navigation easier